### PR TITLE
Request OPTIONS fix to return all allowed methods

### PR DESCRIPTION
**Pull request by michalbundyra**
The integration tests are pulled from zend-expressive-router repository.

See: https://github.com/zendframework/zend-expressive-router/pull/53

### CODE REVIEW

File: composer.json
Comment (geerteltink) [inline composer.json]: I guess this is here for testing only as it should never be merged.

File: composer.json
Comment (geerteltink) [inline composer.json]: Same, just a reminder to change this before merging.

File: composer.json
Comment (geerteltink) [inline composer.json:32]: Can we please stop with all these bugfix requirements all the time? Why is 7.0.2 needed? Do the tests fail with ^7.0?

**Review by geerteltink — DISMISSED** [review]

File: composer.json
Comment (michalbundyra) [reply composer.json]: This PR depends on  zendframework/zend-expressive-router#53 and there is no other way (or I don't know it) to test it than including the repository from my branch with changes. Once  zendframework/zend-expressive-router#53 is merged I'll update this (and also other PRs).

File: composer.json
Comment (michalbundyra) [reply composer.json:32]: For all packages we are always using bugfixes version, why we shouldn't do the same for PHPUnit? It's the latest version available, and it doesn't matter too much as it is just testing framework. I wouldn't bother about it at all.

Comment (weierophinney) [reply composer.json:32]: I'm with @xtreamwayz here; if you're not updating the requirements for a specific reason, there's no need to bump them typically.

(That said, I won't block this PR based on that; just noting that we shouldn't be bumping requirements on every patch, but only when we really need to.)

File: test/IntegrationTest.php
Comment (weierophinney) [inline test/IntegrationTest.php]: I renamed the test case when I merged the patch on zend-expressive-router; it's now `ImplicitMethodsIntegrationTest`.

**Pull request merged**
